installers/launcher-config-windows.yaml

```diff
@@ -33,11 +33,11 @@ updateArchive: null
 updateWindowsRunner: ! '%APPDATA%\Daedalus\Installer.bat'
 
 nodeTimeoutSec: 30
-reportServer: http://report-server.cardano-mainnet.iohk.io:8080
+reportServer: http://report-server.awstest.iohkdev:8080
```

Comment:
@domenkozar @cleverca22 I believe that you have a missing `.io` part here. The `reportServer` url should read: `http://report-server.awstest.iohkdev.io:8080`?

Reply:
yes :(

Reply:
OK. But still - even with the "correct" url: http://report-server.awstest.iohkdev.io:8080 I get error 500 `Internal server error` :((

Reply:
@domenkozar @cleverca22 I see this issue is still not fix and it blocks us from testing of the send-bug-report feature via the installers :(
Can you please help?
<img width="524" alt="screen shot 2018-02-22 at 11 05 45" src="https://user-images.githubusercontent.com/376611/36532284-9316eeb4-17c0-11e8-8f3a-5ef1e958c183.png">

cc @darko-mijic @DominikGuzei

Reply:
@nikolaglumac Fixed - sorry I missed this :(